geo-types/src/geometry/multi_line_string.rs

```diff
@@ -231,6 +231,20 @@ mod test {
     use super::*;
     use crate::{line_string, wkt};
 
+    #[test]
```

Comment:
Don't forget to feature gate theses tests

Reply:
😅